railties/lib/rails/commands/commands_tasks.rb

```diff
@@ -26,20 +30,23 @@ class CommandsTasks # :nodoc:
  runner       Run a piece of code in the application environment (short-cut alias: "r")
 
 All commands can be run with -h (or --help) for more information.
+
 EOT
 
-    COMMAND_WHITELIST = %w(plugin generate destroy console server dbconsole runner new version help test)
+    COMMAND_WHITELIST = %w(plugin generate destroy console server dbconsole runner new version help test rake)
```

Comment:
We still want to phase `rake` out. You see you have added this command so people can query Rake version and help info.

Can't they get the Rake version from their Gemfile.lock?
What does the Rake help info help people with?

I'm leaning towards removing this extra command.